sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubProducer/EventHubProducerLiveTests.cs

```diff
@@ -181,5 +181,302 @@ await using (var producer = client.CreateProducer())
+            {
+                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
+
+                await using (var client = new EventHubClient(connectionString))
+                await using (var producer = client.CreateProducer())
+                {
+                    var events = new[]
+                    {
+                        new EventData(Encoding.UTF8.GetBytes("This is a message")),
+                        new EventData(Encoding.UTF8.GetBytes("This is another message")),
+                        new EventData(Encoding.UTF8.GetBytes("So many messages"))
+                    };
+                    Assert.That(async () => await producer.SendAsync(events), Throws.Nothing);
+                }
+            }
+        }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubProducer" /> is able to
+        ///   connect to the Event Hubs service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ProducerCanSendZeroLengthBatch()
+        {
+            await using (var scope = await EventHubScope.CreateAsync(1))
+            {
+                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
+
+                await using (var client = new EventHubClient(connectionString))
+                await using (var producer = client.CreateProducer())
+                {
+                    var events = new[]
+                    {
+                        new EventData(new byte[0]),
+                        new EventData(new byte[0]),
+                        new EventData(new byte[0])
+                    };
+                    Assert.That(async () => await producer.SendAsync(events), Throws.Nothing);
+                }
+            }
+        }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubProducer" /> is able to
+        ///   connect to the Event Hubs service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        public async Task ProducerCanSendLargeBatch()
+        {
+            await using (var scope = await EventHubScope.CreateAsync(1))
+            {
+                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
+
+                await using (var client = new EventHubClient(connectionString))
+                await using (var producer = client.CreateProducer())
+                {
+                    // Actual limit is 1046520 for a single event
+                    var events = new[]
+                    {
+                        new EventData(new byte[1000000 / 3]),
+                        new EventData(new byte[1000000 / 3]),
+                        new EventData(new byte[1000000 / 3])
+                    };
+                    Assert.That(async () => await producer.SendAsync(events), Throws.Nothing);
```

Comment:
nit:  Space between the `}` and `Assert` for consistency.   :)

Reply:
Done!